### PR TITLE
fix confict , add home , add activity peraturan, detail peraturan, detail pasal and add xml item peraturan 

### PR DESCRIPTION
**Pull request by makhalibagas**

### CODE REVIEW

File: app/src/main/java/com/scc/bukusakuonline/ui/home/HomeFragment.java [hunk 1]
```diff
@@ -24,6 +25,11 @@ public View onCreateView(@NonNull LayoutInflater inflater,
                 ViewModelProviders.of(this).get(HomeViewModel.class);
         View root = inflater.inflate(R.layout.fragment_home, container, false);
 
+<<<<<<< HEAD
+        SearchView searchView = root.findViewById(R.id.sercing);
+        searchView.setQueryHint("Cari Nama Berdasarkan NIS");
+=======
+>>>>>>> b71da54852f63f097f12f5e77698192b877d345c
```
Comment (rsamudragawang) [inline app/src/main/java/com/scc/bukusakuonline/ui/home/HomeFragment.java]: apa ini ?

File: app/src/main/java/com/scc/bukusakuonline/ui/MainActivity.java [hunk 2]
```diff
@@ -26,8 +26,8 @@ protected void onCreate(Bundle savedInstanceState) {
                 R.id.navigation_home, R.id.navigation_history,R.id.navigation_dashboard,R.id.navigation_profile)
                 .build();
         NavController navController = Navigation.findNavController(this, R.id.nav_host_fragment);
-        NavigationUI.setupActionBarWithNavController(this, navController, appBarConfiguration);
```
Comment (rsamudragawang) [inline app/src/main/java/com/scc/bukusakuonline/ui/MainActivity.java]: dihapus?

File: app/src/main/res/layout/activity_detail_pasal.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            android:orientation="horizontal">
+            <ImageView
+                android:layout_width="50dp"
+                android:layout_height="40dp"
+                android:src="@drawable/back"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/roboto"
+                android:text="Pasal 1 Definisi Istilah"
```
Comment (rsamudragawang) [inline app/src/main/res/layout/activity_detail_pasal.xml]: use tools:text

File: app/src/main/res/layout/activity_detail_peraturan.xml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            android:orientation="horizontal">
+            <ImageView
+                android:layout_width="50dp"
+                android:layout_height="40dp"
+                android:src="@drawable/back"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/roboto"
+                android:text="Bab 1 Ketentuan Umum"
```
Comment (rsamudragawang) [inline app/src/main/res/layout/activity_detail_peraturan.xml]: use tools:text

File: app/src/main/java/com/scc/bukusakuonline/ui/home/HomeFragment.java (hunk 1 above)
Comment (Faresapn) [reply app/src/main/java/com/scc/bukusakuonline/ui/home/HomeFragment.java]: wkwk blm di solve conflict

**Pull request closed without merging**